.travis.yml

```diff
@@ -20,7 +20,8 @@ cache:
 
 script:
   - bundle exec rake ci
-
+after_success:
+  - codeclimate-test-reporter
```

Comment:
It doesn't error -- I'm not familiar enough with code climate to see if the results are published per-pull-request. But it works locally